sdks/python/apache_beam/runners/portability/stager.py

```diff
@@ -639,31 +638,59 @@ def _get_python_executable():
       raise ValueError('Could not find Python executable.')
     return python_bin
 
+  @staticmethod
+  def remove_dependency_from_requirements(
```

Comment:
i would add a leading underscore since this is an internal helper, but it's good enough.